gematria/datasets/find_accessed_addrs_exegesis.cc

```diff
@@ -146,7 +146,9 @@ Expected<AccessedAddrs> ExegesisAnnotator::findAccessedAddrs(
     handleAllErrors(std::move(std::get<0>(BenchmarkResultOrErr)),
                     [&](SnippetSegmentationFault &CrashInfo) {
                       MemoryMapping MemMap;
-                      MemMap.Address = CrashInfo.getAddress();
+                      uintptr_t MapAddress =
```

Comment:
You could just clear the last 12 bits of `CrashInfo.getAddress()`, that would be (to me) more clear I think.